corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -386,6 +489,12 @@ def last_for_user(cls, user_id):
 
         return get_last_synclog_for_user(user_id)
 
+    def save(self):
+        save_synclog_to_sql(self)
+
+    def delete(self):
+        delete_synclog(self._id)
```

Comment:
just noticing this now but can't you put these into the `AbstractSyncLog` class and then you won't need them here or in `SimplifiedSyncLog`